### PR TITLE
Bug 1183377: Document additional tracking events.

### PR DESCRIPTION
**Pull request by Osmose**
Fixes problems pointed out by bsmedberg in bug 1183377.

@glogiotatidis r?

### CODE REVIEW

File: docs/data_collection.rst [hunk 1]
```diff
@@ -82,8 +82,9 @@ located at https://snippets-stats.mozilla.org whenever an event occurs that we
 would like to measure. These requests are sampled at a rate of 10%, meaning that
 only 10% of the time an event occurs will a request be made.
 
-Requests sent to snippets-stats.mozilla.org contain the following data in
-addition to the normal data available from an HTTP request:
+Requests sent to snippets-stats.mozilla.org contain the following data (sent as
+JSON in the body of the request) in addition to the normal data available from
```
Comment (glogiotatidis) [inline docs/data_collection.rst]: data are sent as url parameters e.g. `foo.html?snippet_id=12&metric=foo`

**Comment by glogiotatidis** [comment]
besides the json nit, looks good. thanks @Osmose

File: docs/data_collection.rst (hunk 1 above)
Comment (Osmose) [inline docs/data_collection.rst]: what have i done

**Comment by Osmose** [comment]
Crap, so I accidentally merged the branch from #120 rather than the branch for this PR. Luckily, that branch was rebased on this one already, so I just had to revert 72f378d97c93d1d32ee6886f48cc3af49d8e02f4 (reverted in https://github.com/mozilla/snippets-service/commit/efe186d90aadaac8732e66cf655bcd9ae014b9b4) and then add a commit with the JSON->URL params fix (https://github.com/mozilla/snippets-service/commit/6fd860d38670cbfd8355f0a8b4b392cc4375302a).

I'll open a new PR with the privacy changes to replace #120. Sorry for the mixup. :\

**Comment by glogiotatidis** [comment]
Ha I thought you would figure out figure merging due to my duplicate comments, no worries :) Don't forget to open the pull request!

**Pull request closed without merging**